tools/apiview/parsers/cpp-api-parser/vcpkg.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "azure-sdk-for-cpp",
+  "version": "1.5.0",
```

Comment:
Would be great if this could float along to stay most current, but I totatly understand why it is like this.  If you have ideas on how to address that which doesn't make complexity crazy, we should look to fix in the future.

Reply:
It's actually a C&P error - the name and version are bogus and wrong - fixing them.